diff -Nru librsvg-2.2.2.orig/Makefile.am librsvg-2.2.2/Makefile.am
--- librsvg-2.2.2.orig/Makefile.am	Sat Feb  1 22:17:27 2003
+++ librsvg-2.2.2/Makefile.am	Mon Feb  3 14:50:45 2003
@@ -48,6 +48,8 @@
 
 librsvg_2_la_LDFLAGS = -version-info @VERSION_INFO@
 
+librsvg_2_la_LIBADD = $(LIBRSVG_LIBS)
+
 librsvgincdir = $(includedir)/librsvg-2/librsvg
 librsvginc_HEADERS = $(GSFHEADERS) \
 	rsvg.h \
@@ -69,7 +71,6 @@
 DEPS = $(top_builddir)/librsvg-2.la
 LDADDS =$(LIBGSF_LIBS)\
 	$(top_builddir)/librsvg-2.la \
-	$(LIBRSVG_LIBS) \
 	$(POPT_LIBS)
 
 rsvg_SOURCES=test-rsvg.c
