diff -urN aa/librsvg-2.7.2/configure.in librsvg-2.7.2/configure.in
--- aa/librsvg-2.7.2/configure.in	2004-06-09 01:24:14.000000000 +0200
+++ librsvg-2.7.2/configure.in	2004-07-01 14:26:50.435229144 +0200
@@ -297,19 +297,12 @@
 ##################################################
 ##################################################
 
-MOZILLA_CFLAGS=
-if test -z "$MOZILLA_CONFIG"; then
-  AC_PATH_PROG(MOZILLA_CONFIG, mozilla-config, no)
-fi
-if test "x$MOZILLA_CONFIG" != "xno"; then
-  _mozilla_include_dir=`mozilla-config --cflags|sed 's/-I\(.*\) .*/\1/'`
-  MOZILLA_CFLAGS="-I$_mozilla_include_dir/plugin -I$_mozilla_include_dir/java -I$_mozilla_include_dir/nspr"
-else
-  AC_MSG_WARN([mozilla-config not found. Mozilla/Netscape plugin will not be built])
-fi
+PKG_CHECK_MODULES(MOZILLA, mozilla-plugin, have_mozilla=yes, have_mozilla=no)
 build_mozilla_plugin=no
-if test "x$MOZILLA_CFLAGS" != "x"; then
+if test "x$have_mozilla" = "xyes"; then
    build_mozilla_plugin=yes
+else
+  AC_MSG_WARN([mozilla-plugin.pc not found. Mozilla/Netscape plugin will not be built])
 fi
 AM_CONDITIONAL(WITH_MOZILLA,[test "x$build_mozilla_plugin" = "xyes"])
 AC_SUBST(MOZILLA_CFLAGS)
