diff -urN librsvg-2.7.0.orig/moz-plugin/Makefile.am librsvg-2.7.0/moz-plugin/Makefile.am
--- librsvg-2.7.0.orig/moz-plugin/Makefile.am	2004-03-10 07:45:23.477564136 +0100
+++ librsvg-2.7.0/moz-plugin/Makefile.am	2004-03-10 07:45:32.537186864 +0100
@@ -6,7 +6,7 @@
 plugin_LTLIBRARIES = 
 endif
 
-INCLUDES+=$(MOZILLA_CFLAGS) -DBINDIR=\"$(bindir)/\" $(GDK_X11_CFLAGS)
+INCLUDES=$(MOZILLA_CFLAGS) -DBINDIR=\"$(bindir)/\" $(GDK_X11_CFLAGS)
 
 libmozsvgdec_la_SOURCES = moz-plugin.c
 libmozsvgdec_la_LDFLAGS = -module -avoid-version 
